scannerlib/configchecks/file_check_test.go

```diff
@@ -801,12 +806,119 @@ func TestFileExistenceCheckComplianceResults(t *testing.T) {
 	}
 }
 
+func TestFileExistenceCheckPipelinedInput(t *testing.T) {
```

Comment:
These 4 tests can be merged into a single `func TestPipelinedInput` test with table-driven testing - see here for an example: https://github.com/google/localtoast/blob/master/scannerlib/configchecks/file_check_test.go#L369